lib/views/widgets/divider/custom_horizontal_divider.dart

```diff
@@ -0,0 +1,13 @@
+
+import 'package:flutter/material.dart';
+import 'package:rich_text_composer/views/commons/colors.dart';
+
+class CustomHorizontalDivider extends StatelessWidget {
```

Comment:
Can `Divider` widget replace this?

Reply:
No, we need custom it. Divider is not rendering as we expected